src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,142 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import duke.exceptions.DukeException;
+import duke.tasklist.TaskList;
+import duke.tasktypes.Deadline;
+import duke.tasktypes.Event;
+import duke.tasktypes.Task;
+import duke.tasktypes.ToDo;
+import duke.ui.Ui;
+
+/**
+ * Represents a Storage instance of Duke.
+ * A storage instance handles all communication between Duke and local data file.
+ */
+public class Storage {
+
+    private static final String DEFAULT_DIRECTORY = "./data/";
+    private static final String DEFAULT_FILEPATH = "./data/duke.txt";
```

Comment:
I like how your code adhere to coding standard!

Reply:
Thank you!